batavia/utils.js

```diff
@@ -684,3 +684,28 @@ batavia.make_callable = function(func) {
     fn.__python__ = true;
     return fn;
 };
+
+/*************************************************************************
+ * utils
+ *************************************************************************/
+
+batavia.utils = batavia.utils || {};
+batavia.utils.isType = function isType(type, obj) {
+    var clas = Object.prototype.toString.call(obj).slice(8, -1);
+    return obj !== undefined && obj !== null && clas === type;
+};
```

Comment:
isType is used in builtin sorted, taking that out will break the function.

Also instanceof is good enough only for checking custom types with custom constructors.
If you need to check JavaScript type this is totally broken. 
More about this with examples in this article:
http://bonsaiden.github.io/JavaScript-Garden/#types.equality

Should i Still remove isType util and rework sorted function in builtins?